CHANGELOG.md

```diff
@@ -0,0 +1,89 @@
+- CSV is now only returned for completed jobs [#60](https://github.com/Mapotempo/optimizer-api/pull/60)
+- Update fronts accordingly to CSV edits [#61](https://github.com/Mapotempo/optimizer-api/pull/61)
+- Ensure matrix, matrix_id and matrix_index consistency [#62](https://github.com/Mapotempo/optimizer-api/pull/62)
+- Dump filtered problems [#62](https://github.com/Mapotempo/optimizer-api/pull/62)
+- Default `multi_trips` value is now 1 [#47](https://github.com/Mapotempo/optimizer-api/pull/47)
+- Bump VROOM to v1.5.0 [#66](https://github.com/Mapotempo/optimizer-api/pull/66)
+- Reduce memory usage for scheduling problems [#71](https://github.com/Mapotempo/optimizer-api/pull/71)
+- Reduce OR-Tools computation time within scheduling heuristic [#78](https://github.com/Mapotempo/optimizer-api/pull/78)
+- Rework route referents definition within scheduling heuristic, in order to improve routes initialization [#89](https://github.com/Mapotempo/optimizer-api/pull/89)
+- Bump Ruby to 2.5.5 [#97](https://github.com/Mapotempo/optimizer-api/pull/97)
+- Delegate quantity rounding to optimizer-ortools [#98](https://github.com/Mapotempo/optimizer-api/pull/98)
+- Improve Force start & breaks through optimizer-ortools [#98](https://github.com/Mapotempo/optimizer-api/pull/98)
+
+### Removed
+
+- `solver_parameter` from the internal model, it is replaced by `solver` and `first_solution_strategy` parameters
+- Relation `id` was mandatory, but was never used or returned [#31](https://github.com/Mapotempo/optimizer-api/pull/31)
+- Vehicle sort by day index [#31](https://github.com/Mapotempo/optimizer-api/pull/31)
+- `overall_duration` parameter from API as it is not functional [#46](https://github.com/Mapotempo/optimizer-api/pull/46)
+
+### Fixed
+
+- Resolution time when using split_independent or max_split procedures
+- Alternative activities with empties and fills
+- Shift computation for sheduling problems
+- Sheduling heuristic now rejects shipments and rests [#28](https://github.com/Mapotempo/optimizer-api/pull/28)
+- Reject empty problems [#35](https://github.com/Mapotempo/optimizer-api/pull/35)
+- Negative time limits [#35](https://github.com/Mapotempo/optimizer-api/pull/35)
+- Expand multi_trips only once [#36](https://github.com/Mapotempo/optimizer-api/pull/36)
+- Remove data from already started jobs [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Clean interrupted working jobs at startup [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Error when delete operation is invoked immedialty after get [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Coerce `first_solution_strategy` into array [#38](https://github.com/Mapotempo/optimizer-api/pull/38)
+- Avoid unecessary route generation [#46](https://github.com/Mapotempo/optimizer-api/pull/46)
+- Correctly handle consecutive delete operations [#52](https://github.com/Mapotempo/optimizer-api/pull/52)
```

Comment:
I think this problem didn't existed in the previous release, it was a bug introduced and fixed within this release.

Reply:
I would prefer to keep this particular point, as it was not the first we met such issue event if it was hidden in previous releases